draft-fossati-tls-attestation.md

```diff
@@ -456,12 +457,15 @@ attestation_channel_binder = {
 ~~~~
 {: #figure-tls-binder title="Format of TLS channel binder."}
 
-* Nonce is the value provided as a challenge by the relying party.
-* The identity key public fingerprint (ik_pub_fingerprint) is a hash of the
+- Nonce is the value provided as a challenge by the relying party.
+- The identity key public fingerprint (ik_pub_fingerprint) is a hash of the
```

Comment:
I reckon _identity key public fingerprint_ can be trimmed down to _identity key fingerprint_ without loss of precision -- and with a small but sensible increase in readability :-)

```suggestion
- The identity key fingerprint (ik_pub_fingerprint) is a hash of the
```